Octokit.Reactive/Clients/IObservablePullRequestReviewClient.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+using System.Reactive;
+
+namespace Octokit.Reactive
+{
+    /// <summary>
+    /// A client for GitHub's Pull Request Review API.
+    /// </summary>
+    /// <remarks>
+    /// See the <a href="https://developer.github.com/v3/pulls/reviews/">Review API documentation</a> for more information.
+    /// </remarks>
+    public interface IObservablePullRequestReviewClient
```

Comment:
From a "naming things" perspective we have a convention where we name the classes/interfaces as a plural - eg in this case `IObservablePullRequestReviewsClient` if you are able to rename the classes/interfaces and files?